lib/cdo/delete_accounts_helper.rb

```diff
@@ -268,11 +268,8 @@ def remove_census_submissions(email)
     @log.puts "Removing CensusSubmission"
     census_submissions = Census::CensusSubmission.where(submitter_email_address: email)
     csfms = Census::CensusSubmissionFormMap.where(census_submission_id: census_submissions.pluck(:id))
-    ciis = Census::CensusInaccuracyInvestigation.where(census_submission_id: census_submissions.pluck(:id))
-    deleted_cii_count = ciis.delete_all
```

Comment:
unfortunately, we still have to delete this information if a user requests us to. our options are:
1. delete it all now from production DB and redshift
2. use SQL to delete it when requested here, without using the model code

option 2 is usually easier. see other examples of SQL in this file, some written by meg recently